### ISSUE TITLE
Custom background instead of the default white canvas

### ISSUE THREAD

**Issue by raghavtilak** [raghavtilak/Paint#5]

**Comment by infiniteltd** [comment]
I would love to take up this issue.

**Comment by VishnuAmit** [comment]
Hey @raghavtilak. Would love to take this task if its still available for me !

**Comment by raghavtilak** [comment]
> I would love to take up this issue.

Hi @infiniteltd ,
 I would be happy to assign you this issue. Could you please show any relevant experience with Kotlin and Android?

**Comment by raghavtilak** [comment]
> Hey @raghavtilak. Would love to take this task if its still available for me !

Hi @VishnuAmit  ,
 I would be happy to assign you this issue. Could you please show any relevant experience with Kotlin and Android?

**Comment by infiniteltd** [comment]
I have not worked with them yet.

**Comment by Yashraj254** [comment]
hey @raghavtilak by custom background, do you mean a pickable color for canvas background or something else?

**Comment by raghavtilak** [comment]
Yeah, a custom background could be a pickable colour, an image or a gradient. This issue can be split into multiple tasks. Maybe first implementing a pickable solid colour canvas, then adding the functionality to use any device image as a canvas.
